src/modules/esl-carousel/plugin/nav/esl-carousel.nav.arrows.less

```diff
@@ -1,15 +1,11 @@
 /** Arrow navigation styles */
 [esl-carousel-container] {
   --esl-carousel-arrow-size: 40px;
-  --esl-carousel-arrow-offset: clamp(
-    -40px,
-    calc(40px - 40 * 100% / 700),
-    -25px
-  );
+  //add 5px carousel padding
```

Comment:
Oh, I didn't think it's appear here )
Then we might need to declare a custom variable for our extended visibility area.